gxaircom.py

```diff
@@ -0,0 +1,47 @@
+import arrow
+import requests
+
+from winds_mobi_provider import Provider, StationStatus, ureg, Q_, Pressure
+
+
+class GxAircom(Provider):
+    provider_code = "gxaircom"
+    provider_name = "gxaircom.com"
+
+    def process_data(self):
+        self.log.info("Processing GxAircom data...")
+        data = requests.get("http://www.mgenet.at/gxaircom/stations.php", timeout=(self.connect_timeout, self.read_timeout)).json()
+        for station in data:
+            winds_station = self.save_station(
+                provider_id=station["stationId"],
+                short_name=station["stationName"],
```

Comment:
The short names are inconsistent, uppercase names look ugly in UIs:
- GETRONIX
- Schnabelberg

It would be perfect if you can do something, otherwise capitalize only the 1st letter.